src/test/java/kg/jarkyn/cobspecserver/ServerTest.java

```diff
@@ -0,0 +1,38 @@
+package kg.jarkyn.cobspecserver;
+
+import kg.jarkyn.cobspecserver.doubles.ClientDouble;
+import kg.jarkyn.cobspecserver.doubles.ListenerDouble;
+import kg.jarkyn.cobspecserver.doubles.ResponseControllerDouble;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ServerTest {
+
+    private ClientDouble client;
+
+    @Before
+    public void setUp() throws Exception {
+        client = new ClientDouble();
+    }
+
+    @Test
+    public void satisfiesReceivedRequest() {
+        Server server = new Server(new ListenerDouble(client), new ResponseControllerDouble("hello"));
```

Comment:
Not a big thing, but I could see this to merge into some sort of a builder à la `Controller.withResponse("hello")` or something similar.